src/components/EnvoyClusters/EnvoyClusters.vue

```diff
@@ -0,0 +1,129 @@
+<template>
```

Comment:
This is pretty much copy-paste of the second one, is there no easy way to re-use this?

Reply:
I consulted this with @tomaszwylezek and his recommendation was to have separate components.
At first, I tried to generify them but ended up with a gigantic switch block of code. I can see how in the future we will improve each component separately. For example, give a better UI on clusters, or put some charts in the stats tab.

Reply:
I believe this is a good idea for improvement, and as this is the way we are currently doing it everywhere else, I'm fine with it